<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CStatusDialogClass</class>
 <widget class="QDialog" name="CStatusDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>331</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dock Widget Status</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="dockWidgetLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Dock Widget:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="dockWidgetsComboBox">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="statusGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="isClosedCheckBox">
        <property name="text">
         <string>closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isFloatingCheckBox">
        <property name="text">
         <string>floating</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="tabbedCheckBox">
        <property name="text">
         <string>tabbed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="isCurrentTabCheckBox">
        <property name="text">
         <string>is current tab</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="flagsGroupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Feature Flags</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QCheckBox" name="closableCheckBox">
        <property name="text">
         <string>DockWidgetClosable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="movableCheckBox">
        <property name="text">
         <string>DockWidgetMovable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="floatableCheckBox">
        <property name="text">
         <string>DockWidgetFloatable</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="deleteOnCloseCheckBox">
        <property name="text">
         <string>DockWidgetDeleteOnClose</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="customCloseHandlingCheckBox">
        <property name="text">
         <string>CustomCloseHandling</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
